glossary.rst

```diff
@@ -0,0 +1,181 @@
+
+.. _glossary:
+
+Glossary
+========
+
+.. glossary::
+   :sorted:
+
+   conanfile
+      Can refer to either `conanfile.txt` or `conanfile.py` depending on what's the context it is
+      used in.
+
+   conanfile.py
+      The file that defines a Conan recipe that is typically used to create packages, but can be used
+      also to consume packages only (see conanfile.txt). Inside of this recipe, it is defined (among
+      other things) how to download the package's source code, how to build the binaries from those
+      sources, how to package the binaries and information for futures consumers on how to consume
+      the package.
+
+   conanfile.txt
+      It is a simplified version of the `conanfile.py` used only for consuming packages. It defines a
+      list of packages to be consumed by a project and can also define the
+      :ref:`generators<generators>` for the build system we are using and if we want to
+      :ref:`import<imports_txt>` files from the dependencies, as shared libraries, executables or
+      assets.
+
+   recipe
+      Python script defined in a `conanfile.py` that specifies how the package is built from sources,
+      what the final binary artifacts are, the package dependencies, etc.
+
+   reference
+      A package reference is the combination of the package name, version, and two optional fields
+      named user and channel that could be useful to identify a forked recipe from the community with
+      changes specific to your company. It is show in the the form of:
+      ``<name>/<version>@<user>/<channel>``.
```

Comment:
I think that's not an only format of reference, right?
also there is recipe reference and package reference AFAIK